docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+
+To install gRPC tools, run:
+
+```sh
+pip install grpcio-tools
+
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository:
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" Python example:
+  $ cd grpc/examples/python/helloworld
+  ```
+
+
+## Run a gRPC application
+
+From the `examples/python/helloworld` directory:
+
+1. Run the server
+
+   ```sh
+   $ python2.7 greeter_server.py &
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ python2.7 greeter_client.py
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the server for the client to call. Our gRPC service is defined using protocol buffers: you can find out lots more about how this works in [What is gRPC?]() and [gRPC Basics: Python][]. For now all you need to know is that both the server and the client "stub" have a `SayHello` RPC method that takes a `HelloRequest` parameter from the client and returns a `HelloResponse` from the server, and that this method is defined like this:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+Let's update this so that the `Greeter` service has two methods. In your IDE or text editor [what should we recommend, Python people?], open `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Don't forget to save the file!
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/python/helloworld` directory:
+
+```
+$ ./run_codegen.sh
+```
+
+This script invokes the protocol buffer compiler as:
+
+```
+$ protoc -I ../../protos --python_out=. --grpc_out=. --plugin=protoc-gen-grpc=&#96;which grpc&lowbar;python&lowbar;plugin&#96; ../../protos/helloworld.proto</pre>
```

Comment:
This line needs to be updated; it should look something like `python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/helloworld.proto`.

Actually that may be exactly it.